src/components/Favicon/index.jsx

```diff
@@ -0,0 +1,16 @@
+import { Helmet } from 'react-helmet';
+import React, { Component } from 'react';
+
+export default class Favicon extends Component {
+  render() {
```

Comment:
Let's add the following static methods to `Favicon`:

```jsx
static propTypes = {
  state: taskGroupState,
};

static defaultProps = {
  state: null,
};
```

You can define `taskGroupState` in `src/utils/prop-types` right under `taskState`:

```jsx
export const taskGroupState = oneOf([
  'FAILED',
  'COMPLETED',
  'RUNNING',
]);
```